plugin/trino-tpcds/pom.xml

```diff
@@ -42,6 +47,19 @@
             <artifactId>joda-time</artifactId>
         </dependency>
 
+        <!-- used by tests but also needed transitively -->
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>log</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>log-manager</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+
```

Comment:
was master broken? Or are these transitively pulled by plugin-toolkit (and hence scope change)?

Reply:
the latter